jakarta.batch.official.tck/src/main/asciidoc/batch-tck-reference-guide.adoc

```diff
@@ -289,34 +315,42 @@ The list of test cases to run can be customized (for debugging) by modifying the
+The intention here is that the test should not wait forever if something catastrophic occurs causing the job to never complete (or if the porting package SPI â€œwaiterâ€ is never notified for some reason).
+The test also can't end too soon, causing a test failure because the wait was not long enough.
 
 This timeout value can be customized (say, to increase when debugging or decrease to force a faster failure in some cases).
 
 Note that some of the tests (e.g. the chunk tests involving time-based checkpointing) will take at least 15-25 seconds to run on any hardware, so any value less than that for the whole TCK will cause some test failures simply due to timing (and not because of any failure in the underlying Jakarta Batch implementation).
 
-The 900 seconds value, then, was chosen to avoid falsely reporting an error because of timing out too soon, allowing plenty of leeway.	It also facilitates debugging.	It does not, however, provide â€œfast failureâ€ in case of a hang or runaway thread.
+The 900 seconds value, then, was chosen to avoid falsely reporting an error because of timing out too soon, allowing plenty of leeway.
+It also facilitates debugging.
+It does not, however, provide â€œfast failureâ€ in case of a hang or runaway thread.
 
 ===	Building the TCK (optional, for reference):
-The TCK tests can be optionally built from source. However, note that for an implementation to pass the TCK, it must run against the shipped TCK test suite binary as-is (and not against a modified TCK). Still it may be convenient to be able to build the TCK from source for debugging purposes.
 
-== Working with TCK source (debugging, etc.) 
+The TCK tests can be optionally built from source.
+However, note that for an implementation to pass the TCK, it must run against the shipped TCK test suite binary as-is (and not against a modified TCK).
+Still it may be convenient to be able to build the TCK from source for debugging purposes.
+
+== Working with TCK source (debugging, etc.)
 
 For most development/debug use cases it is recommended to refer to the source in the Jakarta Batch TCK] GitHub repository [https://github.com/eclipse-ee4j/batch-tck], and to leverage the Maven automation and artifacts there using the associated documentation.
 
 It should be documented how to use tags/releases, etc. to match the official level tested in the TCK distribution.
 
-It is also possible to use the TestNG `build.xml` script's *compile* target, after setting the *src* property appropriately.   We have paid less attention to this more recently and instead focused on the Maven approach.
+It is also possible to use the TestNG `build.xml` script's *compile* target, after setting the *src* property appropriately.
+We have paid less attention to this more recently and instead focused on the Maven approach.
```

Comment:
Is making a new paragraph just a personal preference  or is this part of some well-known style or formatting you applied?

Reply:
It's not going to create a new paragraph. A new paragraph is created only when there is an empty line between two sentences.

In this case I just split every sentence on it's own line.
To be honest I'd go even further and split every sentence in multiple lines. I find it quite hard to read the file with the current formatting.